tests/phpunit/tests/rest-api/wpRestTemplatesController.php

```diff
@@ -678,4 +684,91 @@ protected function find_and_normalize_template_by_id( $templates, $id ) {
 		return null;
 	}
 
+	/**
+	 * @ticket 56467
+	 * @covers WP_REST_Templates_Controller::create_item
```

Comment:
Add covers.

Reply:
I'm not sure what more to add besides the existing one.. Any suggestions?